spec/govuk/diff/pages/tasks/rakefile_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "spec_helper"
+require "rspec/core/rake_task"
+load "./lib/govuk/diff/pages/tasks/rakefile.rake"
+
+RSpec.describe "rakefile" do
```

Comment:
Solid attempt, but we're missing out on exercising a key part of the functionality in the rake tasks here. Specifically - the loading of a YAML file from an externally set `URI`.

I think a better test would be to stub the value of `ENV.fetch` to be the URI of the fixture file. This then allows you to drop the `TEST` env var.